tests/test_cnn/test_wrappers.py

```diff
@@ -374,3 +374,26 @@ def test_nn_op_forward_called():
         wrapper = Linear(3, 3)
         wrapper(x_normal)
         nn_module_forward.assert_called_with(x_normal)
+
+
+@patch('mmcv.cnn.bricks.wrappers.TORCH_VERSION', (1, 10))
+def test_fx_compatibility():
+    try:
+        from torch import fx
+
+        # ensure the fx trace can pass the network
+        for Net in (MaxPool2d, MaxPool3d):
+            net = Net(1)
+            gm_module = fx.symbolic_trace(net)
+            print(gm_module.code)
+        for Net in (Linear, ):
+            net = Net(1, 1)
+            gm_module = fx.symbolic_trace(net)
+            print(gm_module.code)
+        for Net in (Conv2d, ConvTranspose2d, Conv3d, ConvTranspose3d):
+            net = Net(1, 1, 1)
+            gm_module = fx.symbolic_trace(net)
+            print(gm_module.code)
+    except ImportError:
+        # torch.fx might not be available
+        pass
```

Comment:
```suggestion
from mmengine.utils import digit_version
from mmengine.utils.dl_utils import TORCH_VERSION

@pytest.mark.skipif(
    digit_version(TORCH_VERSION) < digit_version('1.8'),
    reason='torch.fx needs Pytorch 1.8 or higher')
def test_fx_compatibility():
    from torch import fx

    # ensure the fx trace can pass the network
    for Net in (MaxPool2d, MaxPool3d):
        net = Net(1)
        gm_module = fx.symbolic_trace(net)
        print(gm_module.code)
    for Net in (Linear, ):
        net = Net(1, 1)
        gm_module = fx.symbolic_trace(net)
        print(gm_module.code)
    for Net in (Conv2d, ConvTranspose2d, Conv3d, ConvTranspose3d):
        net = Net(1, 1, 1)
        gm_module = fx.symbolic_trace(net)
        print(gm_module.code)
```

Reply:
Is mmengine always available in mmcv?

Reply:
The skip condition is problematic: for pytorch 1.8 and pytorch 1.9, we have `torch.fx`, but pytorch native `MaxPool2d, MaxPool3d` does not support empty input. Therefore, `MaxPool2d, MaxPool3d` wrapped in mmcv will also fail `fx.symbolic_trace`.

Reply:
> Is mmengine always available in mmcv?

Right

Reply:
> The skip condition is problematic: for pytorch 1.8 and pytorch 1.9, we have `torch.fx`, but pytorch native `MaxPool2d, MaxPool3d` does not support empty input. Therefore, `MaxPool2d, MaxPool3d` wrapped in mmcv will also fail `fx.symbolic_trace`.

Got it, We can modify the 1.8 to 1.10.

Reply:
Fixed in [fc883ec](https://github.com/open-mmlab/mmcv/pull/2883/commits/fc883ec85f8bbd08fe934d1fa19a22b7051b3c04).